virtual/activations.go

```diff
@@ -517,16 +567,23 @@ func (a *activatedActor) invoke(
 		return io.NopCloser(bytes.NewBuffer(result.([]byte))), nil
 	}
 
+	tr, err := a._host.BeginTransaction(ctx)
```

Comment:
This is wrong. The test fails without this because the "actor" is actually a "worker" which is not supposed to have access to KV storage. The correct solution here is to pass a "no-op" transaction to the `Invoke` and `InvokeStream` method that just returns an error for any operation the worker tries to perform on the KV store. Otherwise this provides workers with KV storage which is not their intent.

Reply:
See comment in environment/types.go about workers